presentation/backend/src/presentation-backend/PresentationManager.ts

```diff
@@ -296,6 +300,9 @@ export interface PresentationManagerProps {
 
   /** @internal */
   addon?: NativePlatformDefinition;
+
+  /** @internal */
+  getLocalizedString?: (key: string) => string;
```

Comment:
This should probably not be `@internal` - we want to allow consumers supply their own localization function in case the backend wants to respond using languages other than English.